packages/access-api/src/types/provisions.ts

```diff
@@ -1,23 +1,21 @@
 import * as Ucanto from '@ucanto/interface'
 import { ProviderAdd } from '@web3-storage/capabilities/src/types'
 
-export type AlphaStorageProvider = 'did:web:web3.storage:providers:w3up-alpha'
-
 /**
  * action which results in provisionment of a space consuming a storage provider
  */
 export interface Provision<ServiceDID extends Ucanto.DID<'web'>> {
   invocation: Ucanto.Invocation<ProviderAdd>
   space: Ucanto.DID<'key'>
   account: Ucanto.DID<'mailto'>
-  provider: AlphaStorageProvider | ServiceDID
+  provider: ServiceDID
```

Comment:
Nit: I would personally loose the generic and use `Ucanto.DID<'web'>` here. Which I think would make ts-expect-error in file above go away as well.